packages/orchestration/src/orchestration-api.ts

```diff
@@ -234,6 +234,9 @@ export interface OrchestrationAccountI {
   getPublicTopics: () => Promise<Record<string, ResolvedPublicTopic<unknown>>>;
 }
 
+/**
+ * An async-flow suitable for use in contracts using Orchestration
```

Comment:
"async-flow" is a pretty opaque term here.

I suggest moving (most of) the "Orchestration flows" section of the README to here.

Reply:
good idea!